src/forEach.js

```diff
@@ -30,4 +31,6 @@ var _forEach = require('./internal/_forEach');
  *      //-> 7
  *      //-> 8
  */
-module.exports = _curry2(_forEach);
+module.exports = _curry2(function forEach(fn, list) {
+  return _hasMethod('forEach', list) ? list.forEach(fn) : _forEach(fn, list);
```

Comment:
Here is a problem.
> Also note that, unlike `Array.prototype.forEach`, Ramda's `forEach` returns the original
> array. In some libraries this function is named `each`.

But if we dispatch `forEach` to `Map`/`Set`, `undefined` will be returned.
See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/forEach#Description
> `forEach` executes the callback function once for each element in the Map object; it does not return a value.

So does `Set`.

So, we should implement `forEach` in this way, right?
```js
module.exports = _curry2(function forEach(fn, list) {
  _hasMethod('forEach', list) ? list.forEach(fn) : _forEach(fn, list);
  return list;
});
```